web/styles/settings.css

```diff
@@ -381,6 +381,20 @@ td .button {
     }
 }
 
+.list_select {
```

Comment:
I think there exists some way where we can skip this code duplication (it's the same as `.settings_select` on line 370).
I have redefined it because there exists some additional CSS for `settings_select` class at line 362. (That makes sense for the places where it has been used, but in our case, we want to skip that part)

Reply:
You can probably do `.settings_select, .list_select`; but you should also this into a comment, e.g.

`// list_select is similar to settings_select, but without the height styling.`

Reply:
Makes sense.
Made the suggested changes. -- thanks!